src/2.1.cpp

```diff
@@ -0,0 +1,130 @@
+// Copyright 2016 Andrew Vaziri
+// CtCI Question 2.1
+// |||Prompt|||
+// Write code to remove duplicates from an unsorted linked list.
+// |||Questions I might ask|||
+// --Do I have to keep the elements that are not removed in the same order?
+// --Can I say that the list holds integers whiteboard simplicity sake?
+// --Am I given the list by reference or by value? Do I return a list or a
+// pointer to a list?
+// --Must I modify the list I am given in place?
```

Comment:
Yeah I suppose what I should have said rather than "in place" is "without making any other data structures". The solution that meets that requirement is RemoveDups3